src/components/dialogs/dataSources/COVIDHosp.svelte

```diff
@@ -3,17 +3,17 @@
   import { covidHospLocations as regions } from '../../../data/data';
   import SelectField from '../inputs/SelectField.svelte';
   import SelectIssue from '../inputs/SelectIssue.svelte';
-  import { DEFAULT_ISSUE } from '../utils';
+  import { formSelections } from '../../../store';
 
   export let id: string;
 
-  let states = regions[0].value;
-  let issue = DEFAULT_ISSUE;
+  let states = $formSelections.covidHosp.states;
+  let issue = $formSelections.covidHosp.issue;
```

Comment:
do we still need these `states` and `issue` variables, or can we get rid of them the way we did with the `auth` variables in #72 ?